commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -229,3 +235,84 @@ def run(self, args, unknown_args):
         args.playbook = 'deploy_stack.yml'
         unknown_args += ('--tags=users',)
         AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+
+class ManageServices(AnsibleOptions, _AnsiblePlaybookAlias):
+    """
+    example usages
+    1. To restart riak and stanchion only for riakcs
+       only option can be skipped to restart all services which are a part of riakcs
+       i.e riak, riak-cs and stanchion in that order
+        commcare-cloud staging service riakcs restart --only riak stanchion riak-cs
```

Comment:
nit: most of these commands follow the convention that would have us do `--only riak,stanchion,riak-cs` i.e. comma-separated rather than space-separated. Ansible `--limit` for example uses that convention.